metadata-service/factories/src/main/java/com/linkedin/gms/factory/common/RestHighLevelClientFactory.java

```diff
@@ -58,13 +62,17 @@
   @Bean(name = "elasticSearchRestHighLevelClient")
   @Nonnull
   protected RestHighLevelClient createInstance() {
-    RestClientBuilder restClientBuilder;
 
+    System.out.println(
+        String.format("Hey we created it! %s %s %s %s %s", this.host, this.port, this.useSSL, this.username, this.password));
+
+
+    RestClientBuilder restClientBuilder;
     if (useSSL) {
-      restClientBuilder = loadRestHttpsClient(host, port, pathPrefix, threadCount, connectionRequestTimeout, sslContext, username,
+      restClientBuilder = loadRestHttpsClient(host, port, pathPrefix, 1, connectionRequestTimeout, sslContext, username,
```

Comment:
why are we hardcoding to 1?

Reply:
Oops - thanks for catching this - remnant from some debugging !